changelogs/unreleased/fix-netbox-memory-corruption.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
+
+* Fixed memory corruption in netbox. Because of the wrong order of the ffi.gc
+  and ffi.cast calls memory of struct error, which was still used, was freed
```

Comment:
I am afraid users have no idea what is `struct error` and some of them may not known what is `ffi`. Better be more user-friendly in explaining user-visible changes. Here the issue was with any error thrown from any netbox function, could lead to a crash.